apps/src/code-studio/pd/application/teacher/ImplementationPlan.jsx

```diff
@@ -166,12 +154,7 @@ ImplementationPlan.associatedFields = [
 const uniqueRequiredFields = {
   [PROGRAM_CSD]: ['csdWhichGrades'],
   [PROGRAM_CSP]: ['cspWhichGrades', 'cspHowOffer'],
-  [PROGRAM_CSA]: [
-    'csaWhichGrades',
-    'csaHowOffer',
-    'csaAlreadyKnow',
```

Comment:
What's the difference between `uniqueRequiredFields` and the `getDynamicallyRequiredFields`?

Reply:
Ah the dynamically required fields are associated with `meets_minimum_criteria_scores`?

Reply:
`getDynamicallyRequiredFields` is the function on each page (if applicable) that passes in which fields are required given the selected programs (i.e. if the user has not selected CSP then they won't be required to fill in any CSP-specific questions). `uniqueRequiredFields` is just defining on this page which questions are specific to which programs and then `getDynamicallyRequiredFields` is just pushing in said fields given the program.

Reply:
Ah, I see. It'd be nice if we could tie these together more closely so that an update to one doesn't require an update to another. But that's for a future refactor!